### PR TITLE
Feat: Added Category Page

### PR DESCRIPTION
**Pull request by shuntzhere**
Screenshot (30)

### CODE REVIEW

File: rules.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/styles.css" />
+    <title>Paths</title>
+  </head>
+  <body>
+    <div class="rules-container">
+      <nav class="quiz-navbar flex flex-align-center justify-between">
+        <div class="nav-logo flex justify-center">gehraaiyaan</div>
+        <div class="nav-right">
+          <!-- <button class="btn">
```
Comment (vivekmehto) [inline rules.html:15]: remove unused code.

**Review by vivekmehto — APPROVED** [review]
Everything seems good to me.  just a suggestion remove unused code.

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  --color-primary: #5c1466;
+  --color-secondary: #aa2d59;
+  --bg-bright: #3cefff;
+  --bg-reef: #00f7ff;
+  --bg-dark: #4592af;
+
+  --color-white: #fdfdfd;
+  --color-black: #222222;
+  --color-shadow: #cfd9de;
+  /* font families */
```
Comment (vaibhav-mahalle) [inline css/main.css:18]: please remove commented code for better readability of your code.

File: rules.html (hunk 1 above)
Comment (shuntzhere) [reply rules.html:15]: Will remove them while refactoring. Thanks for reviewing.

**Review by vaibhav-mahalle — APPROVED** [review]
Suggested a minor change. Everything else looks good.

**Pull request merged**